browser/src/control/Control.AboutDialog.ts

```diff
@@ -50,6 +50,22 @@ class AboutDialog {
 			.cloneNode(true) as HTMLElement;
 		content.style.display = 'block';
 
+		if (content.querySelector('#js-dialog')) {
+			(content.querySelector('#js-dialog') as HTMLAnchorElement).onclick =
+				function () {
+					app.socket.sendMessage('uno .uno:WidgetTestDialog');
+
+					if (
+						document.getElementById('modal-dialog-about-dialog-box-yesbutton')
+					)
+						(
+							document.getElementById(
+								'modal-dialog-about-dialog-box-yesbutton',
+							) as HTMLButtonElement
+						).click();
+				};
+		}
+
```

Comment:
Observation  =), invoking a click event inside a click function?.

Reply:
When user opens the widget test dialog, "About dialog" stays open. So i wanted to close About dialog for better usability.

But yes it looks suspicious, thanks for the comment :)